README.md

```diff
@@ -1,104 +1,215 @@
-# ntStat
+ntStat is a toolkit that uses Bloom filters to track both *k*-mer count and depth information for use in downstream applications. ntStat also models the *k*-mer count histogram using a mixture model, and infers valuable insights about the genome, sequencing data, and individual *k*-mers, *de novo*.
+
+**Contents**
+- [Installation](#installation)
+  - [From source](#from-source)
+- [Dependencies](#dependencies)
+  - [General dependencies](#general-dependencies)
+  - [C++ libraries](#c-libraries)
+  - [Python packages](#python-packages)
+- [Modules](#modules)
+  - [`count`](#count)
+  - [`filter`](#filter)
+  - [`query`](#query)
+  - [`hist`](#hist)
+- [Credits](#credits)
+- [License](#license)
+
+# Installation
+
+## From source
+
+First, ensure all [Dependencies](#dependencies) are available in the setup environment. Then, install ntStat using `meson`:
+
+```shell
+git clone https://github.com/bcgsc/ntStat.git
+cd ntStat
+meson setup build
+meson install -C build
+```
 
-# Subcommands
+Optional: Add `--destdir PREFIX` to `meson install` to change the installation path prefix to `PREFIX`, containing the `bin` directory for the main executable and `lib` for the modules.
 
-## `filter`
+# Dependencies
+
+## General dependencies
+
+- [Meson](https://mesonbuild.com/Quick-guide.html) and a C++ compiler with C++17 support, only required for installation
+- Python >= 3.9
+- [ntCard](https://github.com/bcgsc/ntCard)
 
-Filters *k*-mers based on their counts and stores the output in a Bloom filter (or counting Bloom filter if `--counts` is set, which will make the output 8x larger). *k*-mers with counts between `-cmin` and `-cmax` (inclusive) will be included in the output.
+## C++ libraries
 
-### Usage
+- [btllib](https://github.com/bcgsc/btllib) >=1.7.3
+- [argparse](https://github.com/p-ranav/argparse)
+- [pybind11](https://github.com/pybind/pybind11)
 
+## Python packages
 ```
-filter [--help] [--version] [-k VAR] [-s VAR] -f VAR [-e VAR] [-b VAR] [-cmin VAR] [-cmax VAR] [--counts] [--long] [-t VAR] -o VAR reads
+kneed
+matplotlib
+numpy
+pandas
+scikit-learn
+scipy
+tabulate
+termplotlit
+tqdm
+```
+
```

Comment:
It would be helpful to have a conda command that shows installing all of your dependencies - just so a user can copy and paste your command (until we have ntStat itself on conda)